src/wallet/wallet2.cpp

```diff
@@ -529,7 +530,8 @@ std::unique_ptr<tools::wallet2> make_basic(const boost::program_options::variabl
 
 boost::optional<tools::password_container> get_password(const boost::program_options::variables_map& vm, const options& opts, const std::function<boost::optional<tools::password_container>(const char*, bool)> &password_prompter, const bool verify)
 {
-  if (command_line::has_arg(vm, opts.password) && command_line::has_arg(vm, opts.password_file))
+  const auto password_file = command_line::get_arg(vm, opts.password_file);
+  if (command_line::has_arg(vm, opts.password) && !password_file.empty())
```

Comment:
This one can also be is_arg_defaulted

Reply:
Sorry about that, had missed that I had used `empty()` there.